docs/source/usersguide/mgxs_library.rst

```diff
@@ -0,0 +1,289 @@
+file.
+
+Similar to the other input file types, the multi-group library is provided in
+the XML_ format.  This library must provide some meta-data about the library
+itself (such as the number of groups and the group structure, etc.) as well as
+the actual cross section data itself for each of the necessary nuclides or
+materials.
+
+.. _XML: http://www.w3.org/XML/
+
+------------------------------------------------
+MGXS Library Specification -- cross_sections.xml
+------------------------------------------------
+
+The multi-group library meta-data is contained within the groups_,
+group_structure_, and inverse_velocities_ elements.
+The actual multi-group data itself is contained within the xsdata_ element.
+
+.. _groups:
+
+``<groups>`` Element
+----------------------------------
+
+The ``<groups>`` element has no attributes and simply provides the number of
+energy groups contained within the library.
+
+  *Default*: None, this must be provided.
+
+.. _group_structure:
+
+``<group_structure>`` Element
+-----------------------------
+
+The ``<group_structure>`` element has no attributes and should be provided as a
+monotonically increasing list of bounding energies, in MeV, for a number of
+groups. To provide proper energy boundaries, the length of the data within the
+``<group_structure>`` element should be one more than the number of groups in
+the problem.  For example, a two-group problem could be specified as:
+
+.. code-block:: xml
+
+    <group_structure> 0.0 0.625E-6 20.0 </group_structure>
+
+*Default*: None, this must be provided.
+
+.. _inverse_velocities:
+
+``<inverse_velocities>`` Element
+--------------------------------
+
+The ``<inverse_velocities>`` element optionally indicates the average
+inverse velocity corresponding to each of the groups in the problem.
+This element should therefore be an array with a length which matches the
+number of groups set in the groups_ element.
+
+*Default*: Should this be needed by the presence of an ``inverse-velocity``
+score in the ``tallies.xml`` file and not provided in this element, OpenMC
+will simply convert the group mid-point energy to an inverse of the velocity
+and use this information for tallying.
+
+.. _xsdata:
+
+``<xsdata>`` Element
+--------------------
+
+The ``<xsdata>`` element contains the nuclide or material-specific meta-data as
+well as the actual cross section data. The following are the
+attributes/sub-elements required to describe the meta-data:
+
+  :name:
+    The name of the microscopic or macroscopic data set.
+
+    *Default*: None, this must be provided.
+
+  :alias:
+    The number of total fission source iterations per batch.
+
+    *Default*: If no alias is provided, it will adopt the value of ``name``.
+
+  :kT:
+    The temperature the data was generated at.
+
+    *Default*: Room temperature, 2.53E-8 MeV
+
+  :fissionable:
+    This element states whether or not the data in question is fissionable.
+    Accepted values are ``true`` or ``false``.
```

Comment:
I generally stick to using quotes to describe data in the text node and using literals to describe the names of elements/attributes. This comment applies to many places.